plugins/janus_audiobridge.c

```diff
@@ -2931,7 +2979,39 @@ static json_t *janus_audiobridge_process_synchronous_request(janus_audiobridge_s
 			g_free(room_id_str);
 		janus_mutex_unlock(&rooms_mutex);
 		goto prepare_response;
-	} else if(!strcasecmp(request_text, "edit")) {
```

Comment:
This should be *after* both `edit` and `destroy`, not before.

Reply:
done